src/client/components/CycleSwitch/CycleSwitch.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { useParams } from 'react-router-dom'
+
+type Props = {
+  components: Record<string, React.FC<{ query?: string }>>
+  defaultKey?: string
+  query?: string
+}
+
+const Placeholder: React.FC<{ query?: string }> = () => {
+  return <div />
+}
+
+const CycleSwitch: React.FC<Props> = (props) => {
+  const { cycleName } = useParams()
+
+  const { components, defaultKey, ...otherProps } = props
```

Comment:
I would also get AssessmentName here, what do you think?

Reply:
Yes. Please check the updated version. Thanks!